articles/custom-domains/additional-configuration.md

```diff
@@ -103,6 +108,21 @@ If you want to use social identity providers with your custom domain, you must u
 ::: warning
 You cannot use [Auth0 developer keys](/connections/social/devkeys) with custom domains.
 :::
+  
+## Configure Google Apps connections
+
+If you want to use Google Apps connections with your custom domain, you must update the Authorized redirect URI in your OAuth Client Settings. In the Google Developer Console, go to **Credentials**, choose your OAuth client in the list, and you will see a settings page with the app Client ID, secret, and other fields. In the **Authorized redirect URIs** field, add a URL in the format `https://<CUSTOM DOMAIN>/login/callback` that includes your custom domain (such as `https://login.northwind.com/login/callback`).
+
+## APIs
+
```

Comment:
This doc https://auth0.com/docs/api-auth/tutorials/verify-access-token should also have this info